code/__DEFINES/mobs.dm

```diff
@@ -131,6 +131,7 @@
 #define SPECIES_SIMIAN "simian" //Monkestation Addition
 #define SPECIES_GOBLIN "goblin" //Monkestation Addition
 #define SPECIES_FLORAN "floran" //Monkestation Addition
+#define SPECIES_NABBER "nabber" //Monkeystation Addition
```

Comment:
this can be put in the ~monkestation defines folder, also while your at it also move the other monke edits, IPC and oozeling are also both edits just unmarked